data/scenarios/Testing/956-GPS.yaml

```diff
@@ -0,0 +1,42 @@
+version: 1
+name: Test GPS receiver device
+description: |
+  Test the `whereami` command and GPS receiver device.
+  https://github.com/swarm-game/swarm/issues/956
+objectives:
```

Comment:
Add a `goal` here that explains what the solution is doing to meet the `condition`

Reply:
Oh, good idea.  While I was at it I also thought of a way to make the test much better --- what do you think now?

Reply:
Yes, using `random` is great.